### PR TITLE
{bio}[foss/2016b] ChimPipe 0.9.5 + BEDTools 2.27.1

### PR DESCRIPTION
**Pull request by SethosII**
This adds ChimPipe with a updated BEDTools.

I can't think of a good `sanity_check_commands` line because printing the help message returns with non-zero exit code and a real example would need additional data.

### CODE REVIEW

**Comment by boegel** [comment]
@SethosII I had to change the target branch to `develop` (was `master`).

Since you have been issuing quite a lot of pull requests lately (thank you for that!), maybe you should take a look a the support EasyBuild has for creating pull requests via `eb --new-pr`, see http://easybuild.readthedocs.io/en/latest/Integration_with_GitHub.html.

Not only will it automatically take care of opening the PR to `develop` rather than `master`, it will also simplify things a bit for you.

File: easybuild/easyconfigs/b/BEDTools/BEDTools-2.27.1-foss-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+homepage = "https://github.com/arq5x/%(namelower)s2"
+description = """The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps
+ and computing coverage. The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF,
+ and SAM/BAM."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://github.com/arq5x/%(namelower)s2/releases/download/v%(version)s/']
+sources = [SOURCELOWER_TGZ]
```
Comment (boegel) [inline easybuild/easyconfigs/b/BEDTools/BEDTools-2.27.1-foss-2016b.eb]: @SethosII This should be `SOURCELOWE_TAR_GZ` instead?

Also, please include a SHA256 checksum.

File: easybuild/easyconfigs/c/ChimPipe/ChimPipe-0.9.5-foss-2016b-Python-2.7.12.eb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ chimeric transcripts and fusion genes from Illumina Paired-End RNA-seq data. It combines junction
+ spanning and paired-end read information to accurately detect chimeric splice junctions at base-pair
+ resolution."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/Chimera-tools/%(name)s/archive']
+sources = ['v%(version)s.tar.gz']
+
```
Comment (boegel) [inline easybuild/easyconfigs/c/ChimPipe/ChimPipe-0.9.5-foss-2016b-Python-2.7.12.eb]: style nitpicking: drop empty line

File: easybuild/easyconfigs/c/ChimPipe/ChimPipe-0.9.5-foss-2016b-Python-2.7.12.eb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+dependencies = [
+    ('BEDTools', '2.27.1'),
+    ('SAMtools', '1.5'),
+    ('BLAST+', '2.6.0', versionsuffix),
+]
+
+sanity_check_paths = {
+    'files': ['%(name)s.sh'],
+    'dirs': ['bin', 'src/awk', 'src/bash', 'tools'],
```
Comment (boegel) [inline easybuild/easyconfigs/c/ChimPipe/ChimPipe-0.9.5-foss-2016b-Python-2.7.12.eb]: Any specific binaries you can check for?

File: easybuild/easyconfigs/b/BEDTools/BEDTools-2.27.1-foss-2016b.eb (hunk 1 above)
Comment (SethosII) [reply easybuild/easyconfigs/b/BEDTools/BEDTools-2.27.1-foss-2016b.eb]: @boegel I changed this.

File: easybuild/easyconfigs/c/ChimPipe/ChimPipe-0.9.5-foss-2016b-Python-2.7.12.eb (hunk 2 above)
Comment (SethosII) [reply easybuild/easyconfigs/c/ChimPipe/ChimPipe-0.9.5-foss-2016b-Python-2.7.12.eb]: @boegel I changed this.

File: easybuild/easyconfigs/c/ChimPipe/ChimPipe-0.9.5-foss-2016b-Python-2.7.12.eb (hunk 3 above)
Comment (SethosII) [reply easybuild/easyconfigs/c/ChimPipe/ChimPipe-0.9.5-foss-2016b-Python-2.7.12.eb]: @boegel I added a specific binary and added a folder inside of `bin`.

**Comment by SethosII** [comment]
@boegel Thanks for the hint on `eb --new-pr`, I will take a look.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2470.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/bd9eb2691582efeea74ffb182b2d6ccd for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
node2129.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/771a584b4091128934055005dc5bf688 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @SethosII!

**Pull request merged**